aries-core/src/js/components/helpers/Anchors/NavLink.js

```diff
@@ -3,20 +3,23 @@ import PropTypes from 'prop-types';
 import { Anchor, Text } from 'grommet';
 
 export const NavLink = ({ active, children, label, size, ...rest }) => {
-  const activeTextColor = active ? 'text' : null;
-  const defaultTextColor = 'text-xweak';
-  const fontWeight = 400;
-  const [textColor, setTextColor] = useState(defaultTextColor);
+  const textColor = 'text-strong';
+  const activeFontWeight = 700;
+  const defaultFontWeight = 400;
+  const [fontWeight, setFontWeight] = useState(defaultFontWeight);
 
   return (
     <Anchor
-      color={activeTextColor || textColor}
-      label={<Text weight={fontWeight}>{label || children}</Text>}
-      onBlur={() => setTextColor(defaultTextColor)}
-      onFocus={() => setTextColor(activeTextColor)}
-      onMouseOut={() => setTextColor(defaultTextColor)}
-      onMouseOver={() => setTextColor(activeTextColor)}
-      size={size}
+      color={textColor}
+      label={
+        <Text size={size} weight={fontWeight}>
+          {label || children}
+        </Text>
+      }
+      onBlur={() => setFontWeight(defaultFontWeight)}
+      onFocus={() => setFontWeight(activeFontWeight)}
+      onMouseOut={() => setFontWeight(defaultFontWeight)}
+      onMouseOver={() => setFontWeight(activeFontWeight)}
```

Comment:
You removed `size={size}` for Anchor, I guess it is because you are assuming it will pass with the spread notation ('...rest')? In this case you need to remove the deconstruction of `size` from NavLink (line #5), otherwise, the size will always be undefined.

Reply:
I actually moved size from <Anchor /> to instead be applied to the <Text /> in the Anchor's label prop, otherwise the size wasn't flowing through as expected.

Reply:
Nice